packages/affiliates/package.json

```diff
@@ -1,23 +1,23 @@
 {
   "name": "@uma/affiliates",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
     "@google-cloud/bigquery": "^5.3.0",
-    "@uma/common": "^2.1.0",
-    "@uma/core": "^2.2.0",
-    "@uma/financial-templates-lib": "^2.2.0",
+    "@uma/common": "^2.2.0",
+    "@uma/core": "^2.3.0",
+    "@uma/financial-templates-lib": "^2.3.0",
+    "@uniswap/v3-core": "1.0.0-rc.2",
```

Comment:
Why adding these now? + `prompt`

Reply:
not sure why this changed bth. @mrice32 any idea?

Reply:
o ye. that's exactly what it did.